26-register_publication/1659-creator_selects_reference_type_contribution_to_journal_and_selects_subtype.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Creator selects Reference type Contribution to journal and selects subtype
+
+    @1659
+    Scenario: Creator sees fields related to Reference subtypes for "Contribution to journal"
+        Given Creator begins registering a Publication in the Wizard
+        When they navigate to the Reference tab
+        And they select the Reference type "Contribution to journal"
+        And they select one of Reference subtypes:
+            | Short communication  |
+            | Letter to the Editor |
+            | Book review          |
+            | Editorial            |
+        Then they see fields:
+            | Search box for Journal |
+            | DOI                    |
+            | Volume                 |
+            | Issue                  |
+            | Pages from             |
+            | Pages to               |
+            | Article number         |
```

Comment:
TODO: legg til validering når formatet på det er greit for PR på Corrigendum/JournalArticle